release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1136,6 +1140,10 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Previously, if you configured the rendezvous IP to the IP address of the host that has a `role` parameter set to `worker` and you generated an ISO image, the Agent-based installer would fail to install the cluster. Now, when you attempt to generate an ISO image based on this configuration, you will receive a validation failure message. On receiving this message, you must update the `rendezvousIP` field in the `agent-config.yaml` file to use the IP of a host with the `master` role. (link:https://issues.redhat.com/browse/OCPBUGS-2088[*OCPBUGS-2088*])
 
+* Previously, the `openshift-installer ` program did not accept the following new regions defined in the `aws-sdk-go` library: `ap-south-2`, `ap-southeast-4`, `eu-central-2`, `eu-south-2`, and `me-central-1`. When you used the program to create the installation configuration file, the program would not list these new regions or accept any manually entry attempts for these regions. Now, the `openshift-installer ` program supports these regions and you can specify them when you create the installation configuration file. (link:https://issues.redhat.com/browse/OCPBUGS-10213[*OCPBUGS-10213*])
```

Comment:
Is "Now" incorrect to use? I'm not seeing any guidance that discourages its use? 
---------------
Updated the rest of the text.

Reply:
Example in the Documentation guidelines GH page:

![Screenshot from 2023-04-28 11-20-49](https://user-images.githubusercontent.com/57954076/235122381-a137228c-0137-4e6d-a852-c3b79ae0b4bf.png)

Reply:
> Is "Now" incorrect to use? I'm not seeing any guidance that discourages its use?
> 
> Updated the rest of the text.

No, I don't think it is incorrect. I just suggested "with this update" because that keeps consistency with other bug's text.